modules/monitoring-example-remote-write-queue-configuration.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="example-remote-write-queue-configuration_{context}"]
+= Example remote write queue configuration
+
+// Set attributes to distinguish between cluster monitoring examples and user workload monitoring examples.
+ifndef::openshift-dedicated,openshift-rosa[]
+:configmap-name: cluster-monitoring-config
+:namespace-name: openshift-monitoring
+:prometheus-instance: prometheusK8s
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+:configmap-name: user-workload-monitoring-config
+:namespace-name: openshift-user-workload-monitoring
+:prometheus-instance: prometheus
+endif::openshift-dedicated,openshift-rosa[]
+
+You can use the `queueConfig` object for remote write to tune the remote write queue parameters. The following is the example configuration of the queue parameters for 
+ifndef::openshift-dedicated,openshift-rosa[]
+default platform monitoring
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+monitoring for user-defined projects
+endif::openshift-dedicated,openshift-rosa[]
+in the `{namespace-name}` namespace.
+
+.Example configuration of remote write parameters
+[source,yaml,subs="attributes+"]
+
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {configmap-name}
+  namespace: {namespace-name}
+data:
+  config.yaml: |
+    {prometheus-instance}:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com" 
+        <endpoint_authentication_credentials>
+        queueConfig:
+          #example parameters #<1>
+----
+<1> TBD
```

Comment:
* capacity: the number of samples to buffer per shard before dropping them.
* minShards: the minimum number of shards, i.e. amount of concurrency.
* maxShards: the maximum number of shards, i.e. amount of concurrency.
* maxSamplesPerSend: the maximum number of samples per send.
* batchSendDeadline: the maximum time a sample will wait in buffer.
* minBackoff: the initial retry delay. It gets doubled for every retry.
* maxBackoff: the maximum retry delay.
* retryOnRateLimit: whether to retry upon receiving a 429 status code from the remote-write storage.
* sampleAgeLimit: drops samples older than the limit.